lib/action/invoke_api_action.dart

```diff
@@ -103,16 +106,23 @@ class InvokeAPIController {
       }
 
       try {
-        // Setting Loading
+        final APIResponse? oldResponse =
```

Comment:
Please test this thoroughly. You assume the old response is in the same Scope which may or may not be true.

Reply:
@vinothvino42 please resolve the conflict. It doesn't look right and things have changed.

Reply:
@vusters I resolved it. Can you review it?